doc/classes/Window.xml

```diff
@@ -325,6 +336,7 @@
 		<member name="content_scale_size" type="Vector2i" setter="set_content_scale_size" getter="get_content_scale_size" default="Vector2i( 0, 0 )">
 		</member>
 		<member name="current_screen" type="int" setter="set_current_screen" getter="get_current_screen" default="0">
+			The screen the window will appear on if the user has more than one display.
```

Comment:
Not only "will appear on", the API doesn't necessarily describe only what happens after you create a node, but also the current state. So this is the screen the window **is** displayed on too.